examples/plots/Four_Panel_Map.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2008-2016 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""
+Four Panel Map
+===============
+
+By reading model output data from a netCDF file, we can create a four panel plot showing:
+
+* 300 hPa heights and winds
+* 500 hPa heights and absolute vorticity
+* Surface temperatures
+* Precipitable water
+"""
+
+###########################################
+
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.gridspec as gridspec
+import matplotlib.pyplot as plt
+import netCDF4
+import numpy as np
+import scipy.ndimage as ndimage
+
+from metpy.cbook import get_test_data
+from metpy.units import units
+
+###########################################
+
+
+# Function used to create the map subplots
+def plot_background(ax):
+    ax.set_extent([235., 290., 20., 55.])
+    ax.coastlines('50m', edgecolor='black', linewidth=0.5)
+    ax.add_feature(states_provinces, edgecolor='black', linewidth=0.5)
+    ax.add_feature(country_borders, edgecolor='black', linewidth=0.5)
+    return ax
+
+
+###########################################
+
+# Open the example netCDF data
+ds = netCDF4.Dataset(get_test_data('gfs_output.nc', False))
+print(ds)
+
+###########################################
+
+# Convert number of hours since the reference time into an actual date
+time_vals = netCDF4.num2date(ds.variables['time'][:].squeeze(), ds.variables['time'].units)
+
+###########################################
+
+# Combine 1D latitude and longitudes into a 2D grid of locations
+lon_2d, lat_2d = np.meshgrid(ds.variables['lon'][:], ds.variables['lat'][:])
+
+###########################################
+
+# Assign units
+vort_500 = ds.variables['vort_500'][0] * units('1/s')  # To plot a e-5 s^-1
```

Comment:
This is a netCDF file...are the units not available as a `units` attribute? (I'm not sure why this is only just now occurring to me)